docs/tools/FindOverlapMatch.md

```diff
@@ -0,0 +1,40 @@
+# FindOverlapMatch
+
+## Introduction
+ 
```

Comment:
This will be fixed in the next version when this get fetched from the source directly, can be ignored for this merge